src/components/FeaturedImage/FeaturedImage.js

```diff
@@ -13,6 +13,7 @@ export default function FeaturedImage({ className, image, ...props }) {
         width={width}
         height={height}
         alt={altText}
+        objectFit="contain"
```

Comment:
I found out that "responsive" layout does distort the images with fixed width/height dimensions.

Reply:
@theodesp Interesting. Should we remove it then in favor or `objectFit`?

Reply:
Keeping `layout="responsive"` works well when we're pulling the image dimensions from WordPress (which we do in most cases). In those cases, there isn't any distortion from `next/image` trying to scale the image because it already has the correct ratio. 

Conversely, in the case of the `Posts` component we are trying to force a square image, regardless of what size the original image is.  We could use `objectFit="cover"` rather than `objectFit="contain"` to achieve this. It should force crop whatever size image we get into a square with just CSS.

https://github.com/vercel/next.js/discussions/18701

Reply:
Ahhhh, I see. Thanks for the clarification. Okay, feel free to disregard this!